observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -55,3 +55,29 @@ You can go to the *Observe* pages in the {product-title} web console to view and
+
+[id="configuring-monitoring-for-user-defined-projects-getting-started_{context}"]
+== Configuring monitoring for user-defined projects: Getting started
+
+As a cluster administrator, you can optionally enable monitoring for user-defined projects in addition to core platform monitoring.
+Non-administrator users such as developers can then monitor their own projects outside of core platform monitoring.
+
+Cluster administrators typically complete the following activities to configure user-defined projects so that users can view collected metrics, query these metrics, and receive alerts for their own projects:
+
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enable user-defined projects].
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign the `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit` cluster roles] to grant non-administrator users permissions to monitor user-defined projects.
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-configure-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign the `user-workload-monitoring-config-edit` role] to grant non-administrator users permission to configure user-defined projects.
+* xref:../../observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc#enabling-alert-routing-for-user-defined-projects[Enable alert routing for user-defined projects] so that developers and other users can configure custom alerts and alert routing for their projects.
+* If needed, configure alert routing for user-defined projects to xref:../../observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc#enabling-a-separate-alertmanager-instance-for-user-defined-alert-routing_enabling-alert-routing-for-user-defined-projects[use an optional Alertmanager instance dedicated for use only by user-defined projects].
+* xref:../../observability/monitoring/managing-alerts.adoc#configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts_managing-alerts[Configure alert receivers] for user-defined projects.
+* xref:../../observability/monitoring/managing-alerts.adoc#applying-a-custom-configuration-to-alertmanager-for-user-defined-alert-routing_managing-alerts[Apply a custom configuration to Alertmanager for user-defined alert routing].
+
+After you enable and configure monitoring for user-defined projects, developers and other users can then perform the following activities to set up and use monitoring for their own projects:
```

Comment:
This piece would be tricky consider we are trying to address different types of users. But to ensure that the conversation style is same, how about something similar to below?
```suggestion
After the monitoring is enabled and configured for user-defined projects, as non-administrator user, you can then perform the following activities to set up and use monitoring for their own projects:
```

Reply:
Good suggestion! Updated to

`After monitoring for user-defined projects is enabled and configured, developers and other non-administrator users can then perform the following activities to set up and use monitoring for their own projects:`